### PR TITLE
Add information:  deleting image

### LINKED ISSUE #12: Add: Information on how to delete a singularity container from system

### PR DESCRIPTION
**Pull request by amanmdesai**
Closes #12

### CODE REVIEW

File: _episodes/02-running-containers.md [hunk 1]
```diff
@@ -180,12 +180,29 @@ with Singularity available.
 > {: .solution}
 {: .challenge}
 
-## Exiting a singularity image
+## Exiting a Singularity image
 
-The `exit` command exits a singularity instance. Note that when exiting from the singularity image all the running processes are killed (stopped).
+The `exit` command exits a Singularity instance. Note that when exiting from the Singularity image all the running processes are killed (stopped).
```
Comment (klieret) [inline _episodes/02-running-containers.md]: Can you rephrase: Note that all running processes ... when exiting from the ....? (clearer that way)

File: _episodes/02-running-containers.md [hunk 2]
```diff
@@ -180,12 +180,29 @@ with Singularity available.
...
 
 ```bash
 Singularity> exit
 ```
 
+## Clean Singularity cache
+
+To remove Singularity image from the cache memory use:
```
Comment (klieret) [inline _episodes/02-running-containers.md]: Missing "a" Singularity image... Also comma before "use". Also I'm confused by the word "memory", because this usually means RAM. Maybe just delete "memory" if that's not what you mean?

File: _episodes/02-running-containers.md [hunk 3]
```diff
@@ -180,12 +180,29 @@ with Singularity available.
...
+
+```bash
+singularity cache clean --name <image-name>
+```
+
+## Deleting Singularity image
+
+To delete a Singularity image use the following command:
```
Comment (klieret) [inline _episodes/02-running-containers.md]: , use

File: _episodes/02-running-containers.md (hunk 2 above)
Comment (klieret) [reply _episodes/02-running-containers.md]: Maybe it'd be best to actually explain what the "cache" is (that'd probably also answer my question).

**Review by klieret — CHANGES_REQUESTED** [review]
There's a few minor comments, but I'm also wondering whether this can be included with the rest of the lesson... Is there one of the images that can be removed by them so that they can actually execute some commands?

File: _episodes/02-running-containers.md (hunk 1 above)
Comment (amanmdesai) [reply _episodes/02-running-containers.md]: Done

File: _episodes/02-running-containers.md (hunk 3 above)
Comment (amanmdesai) [reply _episodes/02-running-containers.md]: Done

**Comment by amanmdesai** [comment]
> There's a few minor comments, but I'm also wondering whether this can be included with the rest of the lesson... Is there one of the images that can be removed by them so that they can actually execute some commands?

@klieret we can do that on the Hello World image itself.

File: _episodes/02-running-containers.md (hunk 2 above)
Comment (amanmdesai) [reply _episodes/02-running-containers.md]: Thanks for this useful comment. 
​
No I didn't mean RAM.

​So when an image is downloaded, it is stored in a cache ​folder ($HOME/.singularity/cache). So the ​`... cache clean ...` command removes the image from cache​ folder​.

I think ​we **need to check whether** `singularity delete` and `singularity cache clean` commands are similar. (This we might also need to specify in the module.)?

**Comment by klieret** [comment]
Sounds good. Is the Hello World thing already in? Then I would directly use that for the commands

**Comment by amanmdesai** [comment]
> Sounds good. Is the Hello World thing already in? Then I would directly use that for the commands

I will modify the things and  include this in epsiode 5. For now I will drop `cache` commands.

File: _episodes/02-running-containers.md (hunk 2 above)
Comment (amanmdesai) [reply _episodes/02-running-containers.md]: For the moment I am removing the information on cache. lets discuss this in a new issue.

**Review by klieret — APPROVED** [review]
